runtime/vm/vmbootlib.c

```diff
@@ -410,7 +410,7 @@ registerBootstrapLibrary(J9VMThread *vmThread, const char *libName, J9NativeLibr
 	char errorBuffer[512] = {0};
 	JavaVMInitArgs *vmInitArgs = (JavaVMInitArgs *) vmThread->javaVM->vmArgsArray->actualVMArgs;
 
-	Trc_VM_registerBootstrapLibrary_Entry(vmThread, libName, libraryPtr);
+	Trc_VM_registerBootstrapLibrary_Entry(vmThread, libName, NULL == libraryPtr ? NULL : *libraryPtr);
```

Comment:
Parentheses around condition (and again on line 424).

Reply:
Added Parentheses.